### ISSUE TITLE
Make flex npm install-able

### ISSUE THREAD

**Issue by Fohlen** [inexorgame/inexor-core#449]
We should make `flex` a globally install-able package via `npm install`
This involves

- convert git submodules into a way they can be configured (like media repositories)
- adding all files to the `files, dictionaries, dictionaries.lib` config variables
- offering config variables for various elements, such as `port, location..`
- adding `inexor` as a `bin` into the `package.json`

That way we could use Flex (and thereby Inexor) in a really simple matter in the future
```
npm install @inexor-game/inexor-flex
inexor-flex # <-- this will start flex and the web server, wohoo

# if you like to have a specific version of flex,
npm install @inexor-game/inexor-flex@VERSION
```

**Comment by Fohlen** [comment]
This will also

- deprecate the `inexor_unix` and `inexor.bat`

**Comment by Croydon** [comment]
How does that deprecate inexor_unix and inexor.bat? We still need one-click files which are executing the start command for users.

**Comment by Fohlen** [comment]
well, in that case you can simply `inexor-flex instance start 31415` from the command line..

**Comment by Fohlen** [comment]
I have thought about your comment again. Providing a desktop shortcut to `inexor-flex instance start 31415` will be much easier / more comfortable for the user than a bash script.

**Comment by aschaeffer** [comment]
Already done for linux: https://github.com/inexorgame/inexor-core/blob/master/dist/linux/inexor.desktop
Who wants to create an ico file for windows?

**Comment by a-teammate** [comment]
whats to do on this one? :)

**Comment by Fohlen** [comment]
@a-teammate still depends on #450 which was previously assigned to @aschaeffer 
I am working on it, however it will take some more time..

**Comment by aschaeffer** [comment]
Good morning :-)

What's the state of this issue? Successfully implemented?

**Comment by Fohlen** [comment]
Yes. Please test wether it works extensively.
